projects/Allwinner/bootloader/firmware

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+[ -n "$ATF_PLATFORM" ] && cp -av $(get_install_dir atf)/usr/share/bootloader/bl31.bin .
+
+CRUST_CONFIG=$($ROOT/$SCRIPTS/uboot_helper $PROJECT $DEVICE $UBOOT_SYSTEM crust_config)
+[ -n "$CRUST_CONFIG" ] && cp -av $(get_install_dir crust)/usr/share/bootloader/scp.bin .
```

Comment:
Can you wrap this into proper if then fi? In current form it returns non-zero return code if there is no crust_config and that stops U-Boot build with error.

Reply:
I guess adding `exit 0` would be cleaner

Reply:
I added a `exit 0` at end of the file, not sure this is what you had in mind :-)

Reply:
it is, thanks!